Minesweeper/Service/FieldService.cs

```diff
@@ -73,7 +87,7 @@ public void CoordinateHasAlreadyBeenUncovered(Coordinate coord)
             }
         }
 
-        public void SetAdjacentCoordinatesToBeUncovered(Coordinate coordinate)
+        public void SetAdjacentCoordinatesToBeUncovered(Coordinate coordinate, CoordinateService coordinateService)
```

Comment:
coordinateService should not pass as a parameter in this function, you can make as a field in this class